kontrakter-felles/main/kotlin/no/nav/tilleggsstonader/kontrakter/dokarkiv/Dokumenttype.kt

```diff
@@ -71,11 +87,33 @@ val Stønadstype.dokumenttyper: Dokumentyper
                     klageVedtaksbrev = Dokumenttype.BOUTGIFTER_KLAGE_VEDTAKSBREV,
                     klageInterntVedtak = Dokumenttype.BOUTGIFTER_KLAGE_INTERNT_VEDTAK,
                 )
+            Stønadstype.DAGLIG_REISE_TSO ->
+                Dokumentyper(
+                    søknad = null,
+                    søknadVedlegg = null,
+                    vedtaksbrev = Dokumenttype.DAGLIG_REISE_TSO_VEDTAKSBREV,
+                    frittståendeBrev = Dokumenttype.DAGLIG_REISE_TSO_FRITTSTÅENDE_BREV,
+                    interntVedtak = Dokumenttype.DAGLIG_REISE_TSO_INTERNT_VEDTAK,
+                    klageVedtaksbrev = Dokumenttype.DAGLIG_REISE_TSO_KLAGE_VEDTAKSBREV,
+                    klageInterntVedtak = Dokumenttype.DAGLIG_REISE_TSO_KLAGE_INTERNT_VEDTAK,
+                )
+            Stønadstype.DAGLIG_REISE_TSR ->
+                Dokumentyper(
+                    søknad = null,
+                    søknadVedlegg = null,
+                    vedtaksbrev = Dokumenttype.DAGLIG_REISE_TSR_VEDTAKSBREV,
+                    frittståendeBrev = Dokumenttype.DAGLIG_REISE_TSR_FRITTSTÅENDE_BREV,
+                    interntVedtak = Dokumenttype.DAGLIG_REISE_TSR_INTERNT_VEDTAK,
+                    klageVedtaksbrev = Dokumenttype.DAGLIG_REISE_TSR_KLAGE_VEDTAKSBREV,
+                    klageInterntVedtak = Dokumenttype.DAGLIG_REISE_TSR_KLAGE_INTERNT_VEDTAK,
+                )
         }
 
 fun Stønadstype.dokumentTypeInterntVedtak(): Dokumenttype =
     when (this) {
         Stønadstype.BARNETILSYN -> Dokumenttype.BARNETILSYN_INTERNT_VEDTAK
         Stønadstype.LÆREMIDLER -> Dokumenttype.LÆREMIDLER_INTERNT_VEDTAK
         Stønadstype.BOUTGIFTER -> Dokumenttype.BOUTGIFTER_INTERNT_VEDTAK
+        Stønadstype.DAGLIG_REISE_TSO -> Dokumenttype.DAGLIG_REISE_TSO_VEDTAKSBREV
+        Stønadstype.DAGLIG_REISE_TSR -> Dokumenttype.DAGLIG_REISE_TSR_VEDTAKSBREV
```

Comment:
Her skal det mappes til internt vedtak som er oppsummeringen av alt som har skjedd i en behandling
```suggestion
        Stønadstype.DAGLIG_REISE_TSO -> Dokumenttype.DAGLIG_REISE_TSO_INTERNT_VEDTAK
        Stønadstype.DAGLIG_REISE_TSR -> Dokumenttype.DAGLIG_REISE_TSR_INTERNT_VEDTAK
```